controllers/configurationpolicy_controller.go

```diff
@@ -1560,6 +1570,30 @@ func (r *ConfigurationPolicyReconciler) determineDesiredObjects(
 				return nil, &scopedGVR, errEvent, nil
 			}
 
+			// Skip adding objects named $SKIP -- This allows users to leverage
+			// templates to further filter results from the objectSelector
+			if desiredObj.GetName() == "$SKIP" {
```

Comment:
I think instead of the `$SKIP`, we wanted a separate template function that could be called at any point to skip the object. This would be a better experience since you could call it when nested down in conditionals unrelated to the name.

Reply:
Ah, indeed. That makes sense--I forgot about that detail. I've removed the `$SKIP` logic.